tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -19,7 +19,7 @@
             result = JSON.parse(data.message).result
             pending_resolve(result);
         } else {
-            pending_reject();
+            pending_reject({ status: data.status, message: data.message });
```

Comment:
Reading the logs, I realize this isn't very human-readable, e.g.

`promise_test: Unhandled rejection with value: object "[object Object]"`

We probably want to return a simple string instead, something like: `${data.status}: ${data.message}`

Reply:
Agreed, done!